server/package.json

```diff
@@ -2,6 +2,8 @@
   "dependencies": {
     "@empirica/admin": "^1.0.0-beta.2",
     "axios": "^0.27.2",
+    "markdown-it": "^13.0.1",
```

Comment:
Do we use both of these dependencies, or is `markdown-it` left over from a test? You can uninstall by using `empirica yarn remove <package>`